assignment/assignment-8-timed/assignment-8.7.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment-8.7</title>
+    </head>
+    <body>
+        <h1>Photo Contest Entry</h1>
+        <form>
+            <label for="fullNameInput">Full Name</label>
+            <input type="text" name="fullName" id="fullNameInput" />
+            <label for="emailInput">Email</label>
+            <input type="email" name="email" id="emailInput" />
+            <br />
+            <br />
+
+            <label for="photoInput">Upload Your Photo:</label>
+            <input
+                type="file"
+                id="photoInput"
+                name="photo"
+                accept="image/*"
+                required
+            />
+            <br />
+            <br />
+
+            <label for="termsConditionsInput"
```

Comment:
better choice for id,for="agreementCheckboxInput"
name="agreement"

Reply:
I like it